api/internal/plugins/loader/loader_test.go

```diff
@@ -78,3 +78,17 @@ func TestLoader(t *testing.T) {
 		}
 	}
 }
+
+func TestLoaderSetPluginConfigWorkingDir(t *testing.T) {
```

Comment:
What's more is that we're testing the whether a deep enough copy was made. Can we change the test name to reflect that? For example, something like `TestDeepCopyPluginConfigWorkingDir`. 

Note that if we change the copy method name, we'll need to change the test name.

Reply:
Yes, apologies, I misread the test code!